go/src/socialapi/workers/api/modules/participant/participant.go

```diff
@@ -45,22 +47,51 @@ func List(u *url.URL, h http.Header, _ interface{}, context *models.Context) (in
 func AddMulti(u *url.URL, h http.Header, participants []*models.ChannelParticipant, context *models.Context) (int, http.Header, interface{}, error) {
 	query := context.OverrideQuery(request.GetQuery(u))
 
+	bo := backoff.NewExponentialBackOff()
+	bo.InitialInterval = time.Millisecond * 50
+	bo.MaxInterval = time.Millisecond * 100
+	bo.MaxElapsedTime = time.Second * 2
```

Comment:
After setting `bo` you'd need to call bo.Reset(), otherwise the InitialInterval [won't get set up](https://sourcegraph.com/github.com/koding/koding@80d10f322aa2d5ae903d6f65421824d253bf9263/-/blob/go/src/vendor/github.com/cenkalti/backoff/exponential.go#L112:1-113:1).